src/styles/todoapp.scss

```diff
@@ -0,0 +1,146 @@
+
+.todoapp {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 24px;
+  font-weight: 300;
+  color: #4d4d4d;
+  margin: 40px 20px;
+
+  &__content {
+    margin-bottom: 20px;
+    background: #fff;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2),
+      0 25px 50px 0 rgba(0, 0, 0, 0.1);
+  }
+
+  &__title {
+    font-size: 100px;
+    font-weight: 100;
+    text-align: center;
+    color: rgba(175, 47, 47, 0.15);
+    -webkit-text-rendering: optimizeLegibility;
+    -moz-text-rendering: optimizeLegibility;
```

Comment:
You don't need to write it, it will be generated automatically after SCCS compilation to CSS.